modules/ROOT/pages/app-declaration.adoc

```diff
@@ -0,0 +1,38 @@
+= Application declaration 
```

Comment:
**issue:** the `application-declaration.adoc` file already exists and is referenced in the navigation. Can it be updated instead of creating a new file?

Reply:
why does it exist? I just changed the name of bc-app-declaration. Did you changed the name of of the previous bc-appplication-declaration without having checked that I created a PR on this page?

Reply:
It exists because I renamed it in #9 and both you and @jeromecambon validated the change prior it was merged on this morning :-) This refactoring should have been done prior other changes. Since it wasn't done, I took care of it myself before we forgot to do it and the documentation was in production.

For SEO and user understanding, it is advised to not abbreviate the names in the URL that's why I preferred to use `application-declaration` instead of `app-declaration`.
That being said, you are free to do it your way if you think better solutions exist.

Reply:
Ok. Thanks.